python/cudf/cudf/core/index.py

```diff
@@ -227,6 +228,14 @@ def step(self):
         """
         return self._step
 
+    @property  # type: ignore
+    @_cudf_nvtx_annotate
+    def end(self):
+        """
+        The absolute last value of the RangeIndex.
+        """
+        return self._end
+
```

Comment:
We don't need to add a new property that doesn't exist in pandas, just use it in defining `max`.
```suggestion
```